app/assets/stylesheets/helpers/_body-with-contents-list.scss

```diff
@@ -0,0 +1,41 @@
+@mixin body-with-contents-list {
+  .body-with-contents-list {
+    position: relative;
+    margin-bottom: $gutter * 1.5;
+  }
+}
+
+.contents-list-link-wrapper .app-c-back-to-top {
+  margin: $gutter-half 0;
+}
+
+// sticky
+.body-with-contents-list .app-c-contents-list {
+  z-index: 0;
+}
+
+.contents-list-link-wrapper.govuk-sticky-element--stuck-to-window {
+  background-color: $panel-colour;
+  margin-left: 0;
+  padding: $gutter-half;
+}
+
+.contents-list-link-wrapper.govuk-sticky-element--stuck-to-window {
```

Comment:
These percentages with negative margins look suspicious, what does this achieve?

Reply:
It's hacky and there's probably a better way:

Content is centred. 
I want a band that is 100% viewport wide with a background colour.
Inside that, I want a link (back to contents link) that lines up exactly with the content. I don't know what that left margin is. If I over-size the band and pull it left by a specific amount, then I know that pushing the link inside back that far will line it up perfectly. 

But there is almost certainly a better way and I'd love to know what it is!